ciao-controller/compute.go

```diff
@@ -1544,7 +1749,7 @@ func createComputeAPI(context *controller) {
 	}).Methods("GET")
 
 	r.HandleFunc("/v2.1/flavors/{flavor}/servers/detail", func(w http.ResponseWriter, r *http.Request) {
-		listServerDetails(w, r, context)
+		listFlavorServerDetail(w, r, context)
```

Comment:
Swagger documents API calls 1 per function, and there were two API calling the same method, that means that one call would not be documented, for this reason, I had to create "secondary functions" that call the primary. This way we can document all API calls